### PR TITLE
[thirdeye] Bump dropwizard to 2.0.33

### PR DESCRIPTION
**Pull request by shounakmk219**
Other dependency version upgrades

- jackson `2.10.2`->`2.10.5`
- swagger-annotation `1.5.22`->`1.6.0`
- dropwizard-swagger `1.3.17-1`->`2.0.0-1`

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -899,6 +889,27 @@
         <groupId>com.smoketurner</groupId>
         <artifactId>dropwizard-swagger</artifactId>
         <version>${swagger.version}</version>
+        <exclusions>
```
Comment (suvodeep-pyne) [inline pom.xml:892]: Is swagger working Ok?

**Comment by cyrilou242** [comment]
I was able to test successfully: 
- swagger
- admin endpoint (in particular metrics)
- bunch of READ calls in-app

File: pom.xml (hunk 1 above)
Comment (shounakmk219) [reply pom.xml:892]: Yes it works fine

**Comment by shounakmk219** [comment]
> I was able to test successfully:
> 
> * swagger
> * admin endpoint (in particular metrics)
> * bunch of READ calls in-app

Thanks a lot Cyril!

**Comment by shounakmk219** [comment]
> ```
> reflections 0.9.11->0.10.2
> mapstruct 1.4.2.Final->1.5.2.Final
> ```
> 
> --> is this specific to the dropwizard upgrade?

reflections is. mapstruct not sure but it was causing problems with IDE. Server run from IDE was giving error due to the mapper implementations generated by mapstruct that's why I upgraded it.
Will handle this separately. Nice catch!

**Comment by shounakmk219** [comment]
Even reflections dependency upgrade was not related to dropwizard upgrade. Reverted it.

**Comment by cyrilou242** [comment]
> Even reflections dependency upgrade was not related to dropwizard upgrade. Reverted it.

cool, let's directly update mapstruct and reflections before dropwizard @shounakmk219

**Pull request merged**